cmd/assetsvc/handler.go

```diff
@@ -52,6 +52,13 @@ type apiResponse struct {
 
 type apiListResponse []*apiResponse
 
+type apiChartCategoryResponse struct {
+	Name  string `json:"name"`
+	Count int    `json:"count"`
+}
```

Comment:
this is the same than your struct `ChartCategory` below, you can just use that one.